Automattic/wp-calypso
# Fix page reloading because of chunk failing to load

This pull request fixes https://github.com/Automattic/wp-calypso/issues/18608 to prevent the `Editor` and `Settings` pages from reloading (a `retry=1` parameter would then be appended to the current url) when users try to access them the first time.

#### Testing instructions

1. Run `git checkout fix/page-reloading` and start your server, or open a [live branch](https://calypso.live/?branch=fix/page-reloading)
2. Open the [`My Site` section](http://calypso.localhost:3000/stats/day/) in a new window
3. Open the `Network` tab in your browser's devtools to keep an eye on `Doc` requests
4. Click the `Settings` menu item in the sidebar
5. Assert that the page is not reloaded (no `retry=1` parameter should be appended to the url)
6. Navigate to any page that displays the `editor`
7. Assert again that this page is not reloaded 

#### Reviews
